frontend/appflowy_flutter/integration_test/desktop/document/document_copy_and_paste_test.dart

```diff
@@ -467,6 +469,52 @@ void main() {
       expect(node.attributes[ImageBlockKeys.url], isNotEmpty);
     });
   });
+
+  testWidgets('paste markdowns', (tester) async {
+    await tester.pasteContent(
+      plainText: '''
+# I'm h1
+## I'm h2
+### I'm h3
+#### I'm h4
+##### I'm h5
+###### I'm h6''',
+      (editorState) {
+        final children = editorState.document.root.children;
+        expect(children.length, 6);
+        for (var i = 0; i < children.length; i++) {
+          final text = children[i].delta!.toPlainText();
+          expect(
+            text,
+            'I\'m h${i + 1}',
+          );
+        }
+      },
+    );
+  });
+
+  testWidgets('paste markdowns as plain', (tester) async {
+    const markdown = '''
+# I'm h1
+## I'm h2
+### I'm h3
+#### I'm h4
+##### I'm h5
+###### I'm h6''';
```

Comment:
Can move this `markdown` const outside of this scope to make it available to the `paste markdowns` test above.

Reply:
All are good suggestions.